### ISSUE TITLE
Turning off fullscreen gives oversized window

### ISSUE THREAD

**Issue by Szunti** [davewx7/citadel#200]
I am running Awesome WM on a laptop with 1366x768 resolution. On fullscreen everything is fine, but if I untick fullscreen I get a window that is much larger than my screen size.

**Comment by galegosimpatico** [comment]
I'm not sure the fix for this will come quick.

While this is geting fixed, you can control the game window width and height when launching windowed.

The exact way to control those might vary a little depending on if you use the standard client or you build your client from sources, but I think in both cases it should reduce to pass these arguments to control the dimensions:

    --width <WIDTH_PIXELS>
    --height <HEIGHT_PIXELS>

You can expect issues if you get far from 16:9 and 16:10 ratios.

Cheers,

**Comment by DDR0** [comment]
Iiiiii mighta just independently fixed it.

**Comment by DDR0** [comment]
"try it now"

**Comment by Szunti** [comment]
No, still wrong, but different. Now the width is less than my resolution, but the height is still huge.

Loading:
![aa-non-full2](https://user-images.githubusercontent.com/1443881/38164829-b3a705b2-350a-11e8-9bd2-5cd3d93f53a7.jpg)

Main screen:
![aa-non-full](https://user-images.githubusercontent.com/1443881/38164832-c227d062-350a-11e8-9165-23c18a634e2b.jpg)

**Comment by Szunti** [comment]
Same. Actually without the huge black areas above and under the main screen it would be a neatly sized window.
